plugins/node/opentelemetry-instrumentation-aws-sdk/src/services/lambda.ts

```diff
@@ -85,7 +85,7 @@ export class LambdaServiceExtension implements ServiceExtension {
     switch (response.request.commandName) {
       case LambdaCommands.Invoke:
         {
-          span.setAttribute(SEMATTRS_FAAS_EXECUTION, response.requestId);
+          span.setAttribute(ATTR_FAAS_INVOCATION_ID, response.requestId);
```

Comment:
here you're changing the attribute key from `faas.execution` to `faas.invocation_id`, which is a breaking change, was that on purpose? 
If so, you need to add a changelog, and also we don't usually just replace, we keep the old at the same time as sending the new one and eventually remove the old one.

Reply:
No, it was not on purpose. I'll push a new commit to keep the old ones.
Is there some indication which can be done to users that old one is deprecated?

Reply:
Reintroduced in [`6333ee3` (#2550)](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2550/commits/6333ee3c1b6c845886e700332df74f66dd5b1bd4)